Quadriga/src/main/webapp/WEB-INF/views/transformation/statusTransformation.jsp

```diff
@@ -0,0 +1,6 @@
+<META HTTP-EQUIV="refresh" CONTENT="5;URL=${pageContext.servletContext.contextPath}/auth/transformation/result/status">
+
+<h1> Transformation is running</h1>
+
+<p>The transformation is RUNNING. Please stay on this page. It will display a download link, once the transformation has finished.
```

Comment:
don't make running all caps (just regular verb spelling). I had a comment to capitalize (just the first character) 'running' in the header, but changed my mind and deleted the comment.